docs/sphinx/reference/dataframe.rst

```diff
@@ -89,6 +89,7 @@ Computations / Descriptive Stats
    DataFrame.var
    DataFrame.sum
    DataFrame.nunique
```

Comment:
We're actually missing entries for `DataFrame.mode` and `Series.mode`, need to create a file under `docs/sphinx/reference/api/eland.DataFrame.mode.rst`:

```rst
eland.DataFrame.mode
====================

.. currentmodule:: eland

.. automethod:: DataFrame.mode

```

and the same for `Series.mode`